frontend/app/src/components/display/AmountDisplay.vue

```diff
@@ -166,6 +128,52 @@ export default class AmountDisplay extends Vue {
     }
     return valueToRender;
   }
+
+  get convertFiat(): boolean {
+    const { ticker_symbol } = this.currency;
+    return !!this.fiatCurrency && this.fiatCurrency !== ticker_symbol;
+  }
+
+  get formattedValue(): string {
+    return this.formatPrice(this.renderValue);
+  }
+
+  get fullValue(): string {
+    return this.convertFiat
+      ? this.convertPrice(this.renderValue).toFormat(2)
```

Comment:
I am trying to think under which circumstances this would display, since we generally don't show tooltips for fiat. The reason this has a hardcoded format is because it represents a fiat currency.

Reply:
Yeah I think the place where we would put the 2 decimals for fiat is within `formatPrice`/ `formatValue`.

Reply:
I am mostly wondering where would that code run. The `fullValue` is displayed in the tooltip, however, there is no tooltip for fiat displays.

Reply:
Right, it wouldn't.

Reply:
I will proceed to remove it in the morning. Unless you can think of a place that this would be used. I cannot think of a case where this tooltip would actually display.